desktop/ui/src/main/java/org/datacleaner/widgets/visualization/JobGraph.java

```diff
@@ -302,6 +330,13 @@ public void paint(Graphics g) {
                 }
 
                 if (subTitle != null) {
+                    
+                    if (error){
+                        g.setColor(WidgetUtils.ADDITIONAL_COLOR_RED_BRIGHT);
```

Comment:
I think making it red will make it way too alarming. This is a quite big message already and it will also feature a "error" icon in the status bar and a icon decoration on the component itself.... Should be enough :-)